### PR TITLE
use plurals tag

### LINKED ISSUE #94: [Assigned] topic_total_session should be plurals text
## Overview (Required)
This text should be plural. so it's better to use `<plurals>` tag.

```xml
<string name="topic_total_session">%dsessions</string>
```

https://github.com/DroidKaigi/conference-app-2018/blob/master/app/src/main/res/values/strings.xml#L59

### PR DESCRIPTION
**Pull request by KoutaMatsushita**
## Issue
- close #94

## Overview (Required)
- Use `<plurals />` at `topic_total_session`

### One Session
EN | JA
:--: | :--:
screenshot_1515832157 | screenshot_1515832213

### Plurals Session
EN | JA
:--: | :--:
screenshot_1515832263 | screenshot_1515832252

### CODE REVIEW

File: app/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -18,7 +18,6 @@
     <string name="session_special_welcome_talk">ウェルカムトーク</string>
     <string name="session_special_lunch">ランチ</string>
     <string name="session_special_party">パーティ</string>
-    <string name="session_fetch_failed">セッションの取得に失敗しました</string>
```
Comment (KoutaMatsushita) [inline app/src/main/res/values-ja/strings.xml]: Sorry, I deleted this resource by mistake. 🙇

**Comment by takahirom** [comment]
I don't know about plurals! It's awesome!!! 👍

**Comment by KoutaMatsushita** [comment]
Please review 🙏

**Comment by konifar** [comment]
Perfect! 💯 LGTM!

**Pull request merged**